web/libs/datamanager/src/sdk/lsf-sdk.js

```diff
@@ -917,12 +917,29 @@ export class LSFWrapper {
     const draftTime = Number(this.task.drafts[0]?.lead_time ?? 0);
     const lead_time = sessionTime + submittedTime + draftTime;
 
+    // StartedAt is the time in which the user first began labeling the task.
+    // It is the earliest time in the task.drafts array or the loadedDate if there are no drafts.
+    // When considering the draft created_at time, the difference between the draft lead_time is subtracted from the current time.
+    // This is done to get an adjusted startedAt time that accounts for the time the user was not actively labeling the task while in a draft state.
+    // This adjustment is necessary to ensure that the started_at time is not before the draftStartedAt time.
+    const draftStartedAt = draft ? new Date(this.task.drafts[0]?.created_at) : undefined;
+
+    let startedAt;
+    if (draftStartedAt) {
+      const draftLeadTime = Number(this.task.drafts[0]?.lead_time ?? 0);
+      const adjustedStartedAt = new Date(Date.now() - draftLeadTime * 1000);
```

Comment:
@ellipsis-dev this would then be calculating in the wrong direction. We are trying to adjust for the time that was not spent labelling by moving the draftStartedAt up, to correct for when a draft may sit for days. The point at which a draft would be converted to an annotation this time frame of when the work was done is adjusted to ensure that calculations based on the annotation started_at will be accurate. The method you proposed would increase the duration of a given annotation, as duration = created_at - started_at.